src/components/NavbarStudyStudio.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, useState} from 'react';
+import classNames from 'classnames';
+import './NavbarStudyStudio.scss';
+import NavbarStudyStudioSVGSelector from './NavbarStudyStudioSVGSelector';
+
+const NavbarStudy: FC = () => {
+    const EMPTY_BUTTON_ID = 0;
+    const [isMenuActive, setIsMenuActive] = useState(false);
+    const [menuType, setMenuType] = useState('study');
+    const [buttonPressed, setButtonPressed] = useState(EMPTY_BUTTON_ID);
+    const items = [
+    {id: 1, value: 'Menu', href: '#!', icon: 'menu', place: 'openclose'},
+    {id: 2, value: 'Close', href: '#!', icon: 'close', place: 'openclose'},
+    {id: 3, value: 'Home', href: '#!', icon: 'home', place: 'study'},
+    {id: 4, value: 'Categories', href: '#!', icon: 'categories', place: 'study'},
+    {id: 5, value: 'Subscription', href: '#!', icon: 'subscription', place: 'study'},
+    {id: 6, value: 'My lessons', href: '#!', icon: 'hat_mylesson', place: 'study'},
+    {id: 7, value: 'Watch later', href: '#!', icon: 'watch_later', place: 'study'},
+    {id: 8, value: 'My lesson', href: '#!', icon: 'my_lesson', place: 'studio'},
+    {id: 9, value: 'Management', href: '#!', icon: 'management', place: 'studio'},
+  ]
+
+let imageWrapperCN = classNames('image__wrapper', !isMenuActive ? 'image__wrapper--active' : '');
```

Comment:
Are you using right classNames? Maybe " classNames('foo', { bar: true }); // => 'foo bar' "

Reply:
It's not work

Reply:
done